app/views/topics/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<section>
+  <h1>Topic: <%= @topic.id %></h1>
```

Comment:
It's generally not a good id to rely on database ids for a user interface. I've worked on several projects where even the database id in the url was obscured.

If you really want topics to be known to users by their unique id then I'd create a feature for this and examine the consequences. 

For now, I'd say remove this from the page.